recipes/antlr4-cppruntime/all/conanfile.py

```diff
@@ -23,133 +28,126 @@ class Antlr4CppRuntimeConan(ConanFile):
+        # We have backport the fix to these old versions.
 
     def validate(self):
-        if str(self.settings.arch).startswith("arm"):
-            raise ConanInvalidConfiguration("arm architectures are not supported")
-            # Need to deal with missing libuuid on Arm.
-            # So far ANTLR delivers macOS binary package.
+        compiler = self.info.settings.compiler
+        compiler_version = Version(compiler.version)
+        antlr_version = Version(self.version)
 
-        compiler = self.settings.compiler
-        compiler_version = tools.Version(self.settings.compiler.version)
-        antlr_version = tools.Version(self.version)
-
-        if compiler == "Visual Studio" and compiler_version < "16":
+        if (compiler == "Visual Studio" and compiler_version < "16") or \
+           (compiler == "msvc" and compiler_version < "1920"):
             raise ConanInvalidConfiguration("library claims C2668 'Ambiguous call to overloaded function'")
             # Compilation of this library on version 15 claims C2668 Error.
-            # This could be Bogus error or malformed Antl4 libary.
+            # This could be Bogus error or malformed Antlr4 library.
             # Version 16 compiles this code correctly.
 
         if antlr_version >= "4.10":
             # Antlr4 for 4.9.3 does not require C++17 - C++11 is enough.
             # for newest version we need C++17 compatible compiler here
+            if compiler.cppstd:
+                check_min_cppstd(self, self._minimum_cpp_standard)
 
-            if self.settings.get_safe("compiler.cppstd"):
-                tools.check_min_cppstd(self, "17")
-
-            minimum_version = self.compiler_required_cpp17.get(str(self.settings.compiler), False)
-            if minimum_version:
-                if compiler_version < minimum_version:
-                    raise ConanInvalidConfiguration("{} requires C++17, which your compiler does not support.".format(self.name))
-            else:
-                self.output.warn("{} requires C++17. Your compiler is unknown. Assuming it supports C++17.".format(self.name))
+            minimum_version = self._compilers_minimum_version.get(str(compiler), False)
+            if minimum_version and compiler_version < minimum_version:
+                raise ConanInvalidConfiguration(f"{self.ref} requires C++{self._minimum_cpp_standard}, which your compiler does not support.")
 
-        if is_msvc(self) and antlr_version == "4.10":
-            raise ConanInvalidConfiguration("{} Antlr4 4.10 version is broken on msvc - Use 4.10.1 or above.".format(self.name))
-
-    def build_requirements(self):
-        if self.settings.os in ("FreeBSD", "Linux"):
-            self.build_requires("pkgconf/1.7.4")
+            if is_msvc(self) and antlr_version == "4.10":
+                raise ConanInvalidConfiguration(f"{self.ref} is broken on msvc - Use 4.10.1 or above.")
```

Comment:
Lets just not package it 🤔 no point in added a dead release

Reply:
Good. I will remove both 4.10.0 and 4.11.0 because they were both dead.